src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -45,14 +45,15 @@
     <PackageReference Include="Microsoft.VisualStudio.Language" Version="$(MicrosoftVisualStudioLanguageVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="$(MicrosoftVisualStudioLanguageIntellisenseVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Language.StandardClassification" Version="$(MicrosoftVisualStudioLanguageStandardClassificationVersion)" />
+    <PackageReference Include="System.Runtime.InteropServices.NFloat.Internal" Version="$(SystemRuntimeInteropServicesNFloatInternalVersion)" />
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
 
     <!-- This project should target net6.0-macos, but due to repeated issues
          provisioning the macos workload on Windows (generally hitting signature
          validation errors on workload dependencies), we are instead manually
          referencing what we need out of the macos SDK. -->
     <PackageDownload Include="Microsoft.macOS.Ref" Version="[$(MicrosoftmacOSRefVersion)]" />
-    <Reference Include="$(NuGetPackageRoot)microsoft.macos.ref\$(MicrosoftmacOSRefVersion)\ref\net6.0\Xamarin.Mac.dll" />
+    <Reference Include="$(NuGetPackageRoot)microsoft.macos.ref\$(MicrosoftmacOSRefVersion)\ref\net6.0\Microsoft.macOS.dll" />
```

Comment:
@sandyarmstrong this is the same change as you're making in https://github.com/dotnet/roslyn/pull/61250

Is this a problem? Or do we need to wait for the 6.0.300 bump in VS Mac?